website/static/js/pages/sharing-page.js

```diff
@@ -40,6 +47,9 @@ $(window).on('load', function() {
     if (typeof cm !== 'undefined') {
       cm.viewModel.onWindowResize();
     }
+    if (typeof arm !== 'undefined') {
+      arm.viewModel.onWindowResize();
```

Comment:
Can you tell me what this is doing?

Reply:
Yes! I was mostly copying the structure of the calls above it, but basically if there are access requests, meaning `arm` is defined, then whenever the window is resized, also call the `AccessRequestManager`'s `onWindowResize` method, which takes care of setting `collapsed()` according to parameters, much like the contributors view model does